services/gafaelfawr/values-roe.yaml

```diff
@@ -0,0 +1,51 @@
+gafaelfawr:
+  imagePullSecrets:
+    - name: "pull-secret"
+  ingress:
+    host: "rsp.lsst.ac.uk"
+  vaultSecretsPath: "secret/k8s_operator/roe/gafaelfawr"
+
+  redis:
+    persistence:
+      enabled: false
+      
+ 
+  config:
+    loglevel: "DEBUG"
+    host: "rsp.lsst.ac.uk"
+    databaseUrl: "postgresql://gafaelfawr@postgres.postgres/gafaelfawr"
+    github:
+      clientId: "10172b4db1b67ee31620"
+
+    # Allow access by GitHub team.
+    groupMapping:
+      "exec:admin":
+        - "lsp-uk-dev"
+      "exec:notebook":
+        - "lsp-uk-dev"
+      "read:workspace":
+        - "lsp-uk-dev"
+      "read:workspace/user":
+        - "lsp-uk-dev"
+      "write:workspace/user":
+        - "lsp-uk-dev"
+      "exec:portal":
+        - "lsp-uk-dev"
+      "exec:user":
+        - "lsp-uk-dev"
+      "read:tap":
+        - "lsp-uk-dev"
+      "read:image":
+        - "lsp-uk-dev"
+
+    initialAdmins:
+      - "stvoutsin"
+
+    errorFooter: |
+      To report problems or ask for help, please open an issue in the
+      <a href="https://github.com/rubin-dp0/Support/issues">GitHub
+      rubin-dp0/Support project</a>.
```

Comment:
You probably don't want this; your users presumably should report issues somewhere other than the DP0 project. You can likely just delete it and use the default for now, but eventually you may want to put your localized bug reporting information here.

Reply:
Removed for now in latest version until we add a page for reporting issues for our site